src/org/centos/pipeline/PackagePipelineUtils.groovy

```diff
@@ -39,8 +39,12 @@ def setDistBranch() {
  * @param artifact ${MAIN_TOPIC}.ci.pipeline.allpackages-${artifact}.<defined-in-README>
  * @return
  */
-def setMessageFields(String messageType, String artifact) {
-    topic = "${MAIN_TOPIC}.ci.pipeline.allpackages-${artifact}.${messageType}"
+def setMessageFields(String messageType, String artifact, Boolean oldFormat) {
+    if (oldFormat) {
+        topic = "${MAIN_TOPIC_OLD}.ci.pipeline.allpackages-${artifact}.${messageType}"
```

Comment:
To publish on new topics, changes will have to be made in https://github.com/fedora-infra/fedmsg_meta_fedora_infrastructure

Reply:
no, you can publish anyway, just the metadata will not translate to anything nice. But thanks for reminder, we are planning to do it.

Reply:
I was added to multiple tickets before because not being in fedmsg_meta gave cert errors on the fedmsg side and they were going to block the messages. I don't know if that is still the case or if what you say is true